README.adoc

```diff
@@ -14,3 +14,145 @@ We would love to hear from you.
 
 If you want to collaborate in the project, we would also love to get your Pull Requests. Before you
 start working on one, please take a look at our link:CONTRIBUTING.adoc[collaboration manual].
+
+== Spring Cloud GCP Pub/Sub
+
+Provides a simplified abstraction to interact with Google Cloud Pub/Sub. Using this abstraction
+allows user code to be independent from the underlying Google Cloud Pub/Sub API semantics. It's
+currently possible to publish messages to a Google Cloud Pub/Sub and to create new topics and
+subscriptions.
+
+You can refer to the `PubSubTemplate` and `PubSubAdmin` classes for more details on the
+functionality provided by this module.
+
+== Spring Cloud GCP Storage
+
+Enables the Spring Resource abstraction to run on Google Cloud Storage.
+
+You can use the `@Value` annotation
+
+[source,java]
+----
+@Value("gs://[YOUR_GCS_BUCKET]/[GCS_FILE_NAME]")
+private Resource file;
+----
+
+or the Spring application context
+
+[source,java]
+----
+SpringApplication.run(...).getResource("gs://[YOUR_GCS_BUCKET]/[GCS_FILE_NAME]");
+----
+
+== Spring Integration channel adapters
```

Comment:
Well, usually we phrase it opposite:

> Spring Integration Channel Adapters for Google Cloud Pub/Sub

The problem here not with the chicken-eggs dilemma, but really who does what. GCP doesn't do anything for Spring Integration, really.